### PR TITLE
More readme/changelog changes 

### PR DESCRIPTION
**Pull request by StephenWeatherford**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Browse and query your MongoDB databases both locally and in the cloud using [_sc
 
 ![Cosmos DB Extension](resources/features.png)
 
+# We've Moved!
+
+You will now find us by clicking in the new Azure view container instead of the file explorer.
+
```
Comment (PrashanthCorp) [inline README.md]: Is there an image here we need to display?

File: README.md [hunk 2]
```diff
@@ -101,6 +102,7 @@ To select which subscriptions show up in the extension's explorer, click on the
 
 - Azure no longer supports gremlin queries on pre-GA graph accounts. If you see the error "Could not find a valid gremlin endpoint for *graph*", then choose "Open Portal" on the graph node and check the "Gremlin Endpoint" in the Overview tab. If it does not take the form of '...[graph-name].***gremlin***.cosmosdb.azure.com...', then you will need to create a new graph account using the Azure portal or the current version of the extension.
 - Graphs are not currently supported with the emulator
+- Viewing/editing tables is not currently supported
```
Comment (PrashanthCorp) [inline README.md]: Nice catch!

File: README.md (hunk 1 above)
Comment (StephenWeatherford) [reply README.md]: Matt didn't want it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
-## 0.7.0 - 2018-05-02
+## 0.7.0 - 2018-05-04
 
 ### Added
```
Comment (EricJizbaMSFT) [inline CHANGELOG.md]: "Execute all comments" should be "Execute all commands"

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
-## 0.7.0 - 2018-05-02
+## 0.7.0 - 2018-05-04
```
Comment (EricJizbaMSFT) [inline CHANGELOG.md]: You know you still have to update the version in package.json, right?

Comment (StephenWeatherford) [reply CHANGELOG.md]: @PrashanthCorp You forget something in your PR? :-)

Comment (StephenWeatherford) [reply CHANGELOG.md]: I'll take care of it .

File: CHANGELOG.md (hunk 3 above)
Comment (StephenWeatherford) [reply CHANGELOG.md]: Nice catch, thanks.

File: CHANGELOG.md (hunk 4 above)
Comment (StephenWeatherford) [reply CHANGELOG.md]: {
	"name": "vscode-cosmosdb",
	"version": "0.7.0",

**Pull request merged**